### DISCUSSION TITLE
Which advantages does this have about the web-standard ActivityPub?

### DISCUSSION THREAD

**Discussion by Jak2k** [bluesky-social/atproto#1782] [Clarifications and "Why?"]
Which advantages does this have about the web-standard ActivityPub?

**Comment by snarfed** [comment]
https://atproto.com/guides/faq#why-not-use-activitypub
https://github.com/bluesky-social/atproto/discussions/1716

**Reply by Jak2k** [reply]
These are no valid reasons.

- When I follow someone, I don't care wether he/she/they can easily port there account.
- How the username is made up doesn't matter. You can easily convert between subdomain and @-notation.
- Large-scale-search and discovery is also possible with ActivityPub. You just need to build a bit on top of existing stuff, instead of reinventing the whole thing. For the same reasons, Tesla didn't build 5-wheel triangular octopus cars with beds in it. They just put another engine into an exisiting concept.

**Reply by DavidBuchanan314** [reply]
It sounds like the reasons don't matter to you personally, which is fair enough, but that doesn't mean the reasons aren't valid. I for one am very excited about account portability.

**Reply by Jak2k** [reply]
But why limit compatibility for many to make portability for a few? Currently, bsky is too centralized. ActivityPub would solve that instantly.

**Reply by DavidBuchanan314** [reply]
It's only centralized because it's not finished yet. I use (and self-host) ActivityPub too, it's not a zero-sum game

**Reply by Jak2k** [reply]
But why not add ActivityPub to bsky? I don't see any harm from this.

**Reply by DavidBuchanan314** [reply]
Well, that's a different question, but the short answer is that their differences are deeper than just syntactical. You might find some of the answers/info here useful: https://github.com/bluesky-social/atproto/discussions/1716
